pkg/controllers/nodeclaim/disruption/drift_test.go

```diff
@@ -56,6 +56,7 @@ var _ = Describe("Drift", func() {
 		})
 		// NodeClaims are required to be launched before they can be evaluated for drift
 		nodeClaim.StatusConditions().SetTrue(v1.ConditionTypeLaunched)
+		Expect(nodeClaim.StatusConditions().Clear(v1.ConditionTypeDrifted)).To(Succeed())
```

Comment:
Was this a flake? Seems unrelated to your change

Reply:
Yeah